### PR TITLE
Add r-clustergeneration

### PR DESCRIPTION
**Pull request by Cashalow**

### CODE REVIEW

**Comment by jdblischak** [comment]
Restarted again.

File: recipes/r-clustergeneration/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  license: GPL (>= 2)
+  summary: We developed the clusterGeneration package to provide functions  for generating random
+    clusters, generating random  covariance/correlation matrices, calculating a separation
+    index (data and population version) for pairs of clusters or cluster distributions,
+    and 1-D and 2-D projection plots to visualize clusters.  The package also contains
+    a function to generate random clusters based on factorial designs with factors such
+    as degree of separation, number of clusters, number of variables, number of noisy
+    variables.
+  license_family: GPL3
+
```
Comment (bgruening) [inline recipes/r-clustergeneration/meta.yaml]: license_file is missing and if you like you can add a few more maintainers.
In general using this script, will help you a little bit: https://github.com/bgruening/conda_r_skeleton_helper

Maintainers are here: https://github.com/bgruening/conda_r_skeleton_helper/blob/master/extra.yaml

File: recipes/r-clustergeneration/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  license: GPL (>= 2)
+  summary: We developed the clusterGeneration package to provide functions  for generating random
+    clusters, generating random  covariance/correlation matrices, calculating a separation
+    index (data and population version) for pairs of clusters or cluster distributions,
+    and 1-D and 2-D projection plots to visualize clusters.  The package also contains
+    a function to generate random clusters based on factorial designs with factors such
+    as degree of separation, number of clusters, number of variables, number of noisy
+    variables.
+  license_family: GPL3
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```
Comment (bgruening) [inline recipes/r-clustergeneration/meta.yaml]: Is something like this possible? https://github.com/conda-forge/staged-recipes/pull/4394/files#diff-110e9d3bd5ca731f3ee4aaf23c056bc9R45

Comment (Cashalow) [reply recipes/r-clustergeneration/meta.yaml]: ah yes sorry I hadn't checked your skeleton helper in details.

**Comment by Cashalow** [comment]
hello @conda-forge/staged-recipes, could this be reviewed and added please ? thanks !

File: recipes/r-clustergeneration/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  license_file: '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'  # [unix]
+  license_file: '{{ environ["PREFIX"] }}\R\share\licenses\GPL-3'  # [win] 
+
+extra:
+  recipe-maintainers:
+    - Cashalow
+    - johanneskoester
+    - bgruening
+    - daler
+    - jdblischak
```
Comment (scopatz) [inline recipes/r-clustergeneration/meta.yaml]: Does everyone here want to be a maintainer?

Comment (Cashalow) [reply recipes/r-clustergeneration/meta.yaml]: I thought @bgruening advised me to add these people as maintainers above but I can remove them if that's a problem.

Comment (scopatz) [reply recipes/r-clustergeneration/meta.yaml]: Oh true!

**Pull request merged**